src/app/components/team/team.component.html

```diff
@@ -1,8 +1,8 @@
 <article>
     <img src="assets/pictures/{{ team.picture }}" alt="{{ team.name }}" />
-    <h3 >{{ team.name }}</h3>
-    <h3>E-mail :</h3>
-    <h3>{{ team.email }}</h3>
+    <h1 >{{ team.name }}</h1>
```

Comment:
Consider the following example as a good practice for html pages : 
ONE H1 for the main title of the page
as much h2 as you want (till there is one H1)
as much h3 as you want (till there is t least one h2)
and so on.....

In this case use h2 for the team name but email is not a title

And use only one H1 in the about us page